cmd/cca/README.md

```diff
@@ -0,0 +1,52 @@
+# Claims Template
+
+```json
+{
+  "cca-platform-token": {
+    // CCA Platform claims
```

Comment:
Does the comments allowed like that in the JSON templates. I thing, we do not need them here anyway!

Reply:
This is just a very sketchy WIP.  There's a specific point in #15 to provide documentation.